samples/matter/lock/README.rst

```diff
@@ -536,6 +641,111 @@ After building the sample and programming it to your development kit, complete t
+#. Add the example ``Home`` door lock user:
+
+   .. code-block:: console
+
+      ./chip-tool doorlock set-user 0 2 Home 123 1 0 0 10 1 --timedInteractionTimeoutMs 5000
+
+   This command creates a ``Home`` user with a unique ID of ``123`` and an index of ``2``.
+   The new user's status is set to ``1``, and both its type and credential rule to ``0``.
+   The user is assigned to the door lock cluster residing on endpoint ``1`` of the node with ID ``10``.
+
+#. Set the example ``week-day`` schedule using the following command:
+
+   ./chip-tool doorlock set-week-day-schedule WeekDayIndex UserIndex DaysMask StartHour StartMinute EndHour EndMinute destination-id endpoint-id
+
+   Where:
+
+   * ``WeekDayIndex`` is an index of the new schedule, starting from ``1``.
+     The maximum value is defined by the :kconfig:option:`CONFIG_LOCK_MAX_WEEKDAY_SCHEDULES_PER_USER` kconfig option.
+   * ``UserIndex`` is the user index defined in the previous step, starting from ``1``.
+   * ``DaysMask`` is a bitmap of numbers of the week days starting from ``0`` as a Sunday and finishing at ``6`` as a Saturday.
+     For example, to assign this schedule to Tuesday, Thursday and Saturday you need to provide ``84`` because it is equivalent to the ``01010100`` bitmap.
+   * ``StartHour`` is the starting hour for the week day schedule.
+   * ``StartMinute`` is the starting minute for the week day schedule.
+   * ``EndHour`` is the ending hour for the week day schedule.
+   * ``EndMinute`` is the ending hour for the week day schedule.
+   * ``node-id`` is the device node ID.
+   * ``endpoint-id`` is the Matter door lock endpoint, in this sample assigned to ``1``.
+
+   For example, use the following command to set a ``week-day`` schedule for Tuesday, Thursday and Saturday to start at 7:30 AM and finish at 10:30 AM dedicated for user with ID ``1``:
+
+   .. code-block:: console
+
+      ./chip-tool doorlock set-week-day-schedule 1 1 84 7 30 10 30 1 1
+
+
+#. Set the example ``year-day`` schedule using the following command:
+
+   ./chip-tool doorlock set-year-day-schedule YearDayIndex UserIndex LocalStartTime LocalEndTime node-id endpoint-id
+
+   Where:
+
+   * ``YearDayIndex`` is an index of the new schedule, starting from ``1``.
+     The maximum value is defined by the :kconfig:option:`CONFIG_LOCK_MAX_YEARDAY_SCHEDULES_PER_USER` kconfig option.
+   * ``UserIndex`` is the user index defined in the previous step, starting from ``1``.
+   * ``LocalStartTime`` is the starting time in Epoch Time.
+   * ``LocalEndTime`` is the ending time in Epoch Time.
+   * ``node-id`` is the device node ID.
+   * ``endpoint-id`` is the Matter door lock endpoint, in this sample assigned to ``1``.
+
+   Both ``LocalStartTime`` and ``LocalEndTime`` are in seconds with the local time offset based on the local timezone and DST offset on the day represented by the value.
+
+   For example:
```

Comment:
I'd expand this to a full sentence that explains the values used, like was done for `Week-day`.

Reply:
Done